src/pages/RequestView/RequestView.tsx

```diff
@@ -126,45 +157,42 @@ const RequestView = () => {
                       <ExpandMoreIcon />
                     )}
                   </IconButton>
-                </Typography>
-                <Typography variant="h6">Outputs</Typography>
-                <Box
-                  border={1}
-                  borderColor="lightgrey"
-                  bgcolor="whitesmoke"
-                  borderRadius="borderRadius"
-                >
-                  <Box p={2}>{outputFormatted(request)}</Box>
-                </Box>
-              </Box>
+                </CardActions>
+                <Divider />
+                <CardContent>
+                  {outputFormatted(request, theme, showAsRawData)}
+                </CardContent>
+              </Card>
             ) : null}
             {!expandOutput ? (
```

Comment:
I really like the way you implemented these expanding cards. Very clever and elegant.